openedx/features/student_account/forms.py

```diff
@@ -76,6 +84,21 @@ def clean_organization_name(self):
                 return org_name
         return org_name
 
+    def clean_g_recaptcha(self):
```

Comment:
Are we sure this cleaner is being called? becuase field name is not consistent with cleaner name.